djstripe/models.py

```diff
@@ -541,20 +542,95 @@ def _attach_objects_hook(self, cls, data):
         if subscription:
             self.subscription = subscription
 
+    def _attach_objects_post_save_hook(self, cls, data):
+        # InvoiceItems need a saved invoice because they're associated via a
+        # RelatedManager, so this must be done as part of the post save hook.
+        cls._stripe_object_to_invoice_items(InvoiceItem, data, self)
```

Comment:
OK, done this - Actually had misread this earlier and thought it meant something else.